gam/clustering.py

```diff
@@ -4,13 +4,16 @@
 FastPAM1 from: https://arxiv.org/pdf/2008.05171.pdf
 Bandit PAM from: https://arxiv.org/pdf/2006.06856.pdf
 """
+import dask.array as da
 import math
 import sys
 import time
 from copy import deepcopy
 
 import matplotlib.pyplot as plt
 import numpy as np
+from sklearn.metrics import pairwise_distances
+from dask_ml.metrics.pairwise import pairwise_distances as dask_pairwise_distances
```

Comment:
The two `pairwise_distances` here will conflict. The import from `dask_ml` will override the `sklearn` import. Would be good to use `as` to name each pairwise for clarity and to avoid conflicts like this.